config/v1/types_infrastructure.go

```diff
@@ -101,6 +101,12 @@ type InfrastructureStatus struct {
 	// +kubebuilder:default=HighlyAvailable
 	// +kubebuilder:validation:Enum=HighlyAvailable;SingleReplica
 	InfrastructureTopology TopologyMode `json:"infrastructureTopology"`
+
+	// cpuPartitioning expresses that CPU partitioning is a currently enabled feature in the cluster.
```

Comment:
This can only be set once in the cluster and never changed, correct?  I think I remember that restriction from the original enhancement.

Reply:
Right, I think the only special instance where we might change is that one time event during a Single Node upgrade from 4.11 -> 4.12